packages/react-data-grid/src/Cell.js

```diff
@@ -317,13 +315,21 @@ class Cell extends React.PureComponent {
     if (treeDepth > 0 && isExpandCell) {
       cellDeleter = <ChildRowDeleteButton treeDepth={treeDepth} cellHeight={this.props.height} siblingIndex={this.props.expandableOptions.subRowDetails.siblingIndex} numberSiblings={this.props.expandableOptions.subRowDetails.numberSiblings} onDeleteSubRow={this.onDeleteSubRow} isDeleteSubRowEnabled={isDeleteSubRowEnabled} />;
     }
+
+    const tooltip = this.props.tooltip ? (<span className="cell-tooltip-text">{this.props.tooltip}</span>) : null;
+
+    const classes = joinClasses('react-grid-Cell__value',
```

Comment:
I think we need to move the tooltip outside the grid otherwise there will be issues similar to editors. Currently if the cell has tooltip then a new div is rendered and positioned. I propose the following solution instead

```
return (
      <div
        {...this.getKnownDivProps()}
        className={className}
        style={style}
        {...events}
        ref={this.setCellRef}
      >
        {cellActionButtons}
        {cellExpander}
        { tooltip ? (<TooltipTrigger placement="left" overlay={tooltip}> {cellContent}<TooltipTrigger > : cellContent }
      </div>
    );
```

`TooltipTrigger` can render the tooltip in a portal and position it